src/app/components/return-button/return-button.component.ts

```diff
@@ -9,8 +9,13 @@ import { Path } from '../../app.paths';
   templateUrl: './return-button.component.html',
   styleUrl: './return-button.component.css'
 })
-export class ReturnButtonComponent {
+export class ReturnButtonComponent implements OnInit{
   private router = inject(Router);
+  private detectorRef = inject(ChangeDetectorRef);
+
+  public ngOnInit(): void {
+    this.detectorRef.detach();
```

Comment:
Interesting. 
I've never seen someone who uses this.
Why do you need this?

Reply:
this is a just a simple button with no changes in the view, thus there is no reason for it to be checked for changes